.github/workflows/maven_ubuntu.yml

```diff
@@ -21,4 +21,4 @@ jobs:
       with:
         java-version: '14.0.1'
     - name: Build with Maven
-      run: mvn -B package --file pom.xml
+      run: mvn -e -B clean test --file pom.xml
```

Comment:
is "-B" necessary ? i feel its redundant. it's already a non interactive command

Reply:
Done.